src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -90911,84 +90878,92 @@ INSERT INTO tab_event_conditions VALUES(5938,'EVENT','ParticleBinDissolvedEventU
+INSERT INTO tab_known_transporters VALUES('ABCC2','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('ABCC3','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('ABCC4','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('ABCC4','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('ABCC5','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('ABCC6','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('ABCG2','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC10A1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC21A1','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC22A1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A1','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A11','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC22A2','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A2','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A4','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC22A4','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC22A5','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC22A5','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC22A6','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A6','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A7','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A8','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A8','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC22A9','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLC47A1','Human','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLCO1A1','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1A2','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1A3','Rat','Efflux');
+INSERT INTO tab_known_transporters VALUES('SLCO1A4','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1B1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1B2','Rat','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO1B3','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO2B1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLCO4C1','Human','Influx');
+INSERT INTO tab_known_transporters VALUES('SLC47A2','Human','Efflux');
+CREATE TABLE tab_known_transporters_containers
+(
+    gene           text,
+    species        text,
+    container_type text,
+    container_name text,
+    transport_type text,
+    membrane       text,
+    constraint tab_known_transporters_containers_pk
+        primary key (gene, species, container_type, container_name),
+    constraint tab_known_transporters_containers_tab_known_transporters_gene_species_fk
+        foreign key (gene, species) references tab_known_transporters (gene, species)
+);

```

Comment:
Missing in the table definition:
* Foreign Key `{container_type, container_name} => tab_container_names`
* **transport_type**: constraint with allowed types (_Influx_, _Efflux_, ...)
  * Actually it would be better to define a new table "**tab_active_transport_types**" with just 1 column "**transport_type**" 
  (with foreign key `tab_active_transport_types.transport_type => tab_process_types.process_type`)
    And then define foreign key `tab_known_transporters_containers.transport_type => tab_active_transport_types`
* membrane: constraint (_Apical_, _Basolateral_)

If you want, we can merge as is and I will add the FKs/Constraints after that.

Reply:
Yes please. I just copied the table that we had before so it somewhat should have taken over the relationship. I don't think we had constraints on columns at all

Reply:
Membrane is also brain barrier etc.